Octokit/Clients/IUsersClient.cs

```diff
@@ -54,6 +54,13 @@ public interface IUsersClient
         /// <returns>A <see cref="User"/></returns>
         Task<User> Update(UserUpdate user);
 
+        /// <summary>
+        /// Returns list of all <see cref="User"/>, in the order that they signed up on GitHub after specified ID.
+        /// </summary>
+        /// <param name="since">ID after which the returned list begin</param>
+        /// <returns>A list of <see cref="User"/> after specified ID</returns>
+        Task<IReadOnlyPagedCollection<User>> GetAll(string since);
```

Comment:
This is almost correct - we have `IReadOnlyList<T>` here which is just what we need for this situation. I've updated this and pushed a commit.